routers/api/v1/repo/collaborators.go

```diff
@@ -349,3 +351,54 @@ func GetAssignees(ctx *context.APIContext) {
 	}
 	ctx.JSON(http.StatusOK, convert.ToUsers(ctx, ctx.Doer, assignees))
 }
+
+// GetAllContributorsStats retrieves a map of contributors along with their weekly commit statistics
+func GetAllContributorsStats(ctx *context.APIContext) {
```

Comment:
to add an api is nice ... and I would let it stay in this pull ...

... but for our web interface we have different routes.

just add a second endpoint for similar to this ... you change error handling there a bit (as you know that this endpoint should only be on non-Empty-Repos (we should have middleware in this case))

Reply:
Sorry, I am reading this comment over an over and I can't understand. What do you want me to do exactly? Add a new endpoint? What for? Also I didn't understand the thing you said about non-empty repos.

Reply:
Haha ok ...

We explizite diveded tje API from our WebUI as we can break/refactor stuff there as we want.

So using the API for the frontend should not be done.

This does not mean you should remove the API endpoint, its usefull.

But for your vue code, you need to untroduce a new route

Give me som sec to lookup the code :)

Reply:
https://github.com/go-gitea/gitea/pull/25439/files#r1238863069

Reply:
OK, done.

Reply:
thanks